pkg/controller/volume/attachdetach/attach_detach_controller.go

```diff
@@ -127,18 +128,21 @@ func NewAttachDetachController(
 	// dropped pods so they are continuously processed until it is accepted or
 	// deleted (probably can't do this with sharedInformer), etc.
 	adc := &attachDetachController{
-		kubeClient:  kubeClient,
-		pvcLister:   pvcInformer.Lister(),
-		pvcsSynced:  pvcInformer.Informer().HasSynced,
-		pvLister:    pvInformer.Lister(),
-		pvsSynced:   pvInformer.Informer().HasSynced,
-		podLister:   podInformer.Lister(),
-		podsSynced:  podInformer.Informer().HasSynced,
-		podIndexer:  podInformer.Informer().GetIndexer(),
-		nodeLister:  nodeInformer.Lister(),
-		nodesSynced: nodeInformer.Informer().HasSynced,
-		cloud:       cloud,
-		pvcQueue:    workqueue.NewNamedRateLimitingQueue(workqueue.DefaultControllerRateLimiter(), "pvcs"),
+		kubeClient:        kubeClient,
+		pvcLister:         pvcInformer.Lister(),
+		pvcsSynced:        pvcInformer.Informer().HasSynced,
+		pvLister:          pvInformer.Lister(),
+		pvsSynced:         pvInformer.Informer().HasSynced,
+		podLister:         podInformer.Lister(),
+		podsSynced:        podInformer.Informer().HasSynced,
+		podIndexer:        podInformer.Informer().GetIndexer(),
+		nodeLister:        nodeInformer.Lister(),
+		nodesSynced:       nodeInformer.Informer().HasSynced,
+		csiDriverInformer: csiDriverInformer,
+		csiDriverLister:   csiDriverInformer.Lister(),
```

Comment:
feature gate informers on the CSIDriver gate like this: https://github.com/kubernetes/kubernetes/blob/83ff0f6c64485ffe491d5116596072f466be1bd5/pkg/controller/volume/attachdetach/attach_detach_controller.go#L146

Reply:
another change necessary when feature gated: https://github.com/kubernetes/kubernetes/pull/74835/commits/41b3579345ff761a54ec56e2afc2d2d2ce6f56e5#diff-72e66cba2ced392dc0f9cd10c424d6d0R327

Reply:
Sure.